cmd/context/use.go

```diff
@@ -226,17 +226,17 @@ func getContext(ctxOptions *Options) (string, error) {
 	return oktetoContext, nil
 }
 
-func setSecrets(secrets []types.Secret) {
-	for _, secret := range secrets {
-		value, exists := os.LookupEnv(secret.Name)
+func exportPlatformVariablesToEnv(variables []env.Var) {
+	for _, v := range variables {
+		value, exists := os.LookupEnv(v.Name)
 		if exists {
-			if value != secret.Value {
-				oktetoLog.Warning("$%s secret is being overridden by a local environment variable by the same name.", secret.Name)
+			if value != v.Value {
+				oktetoLog.Warning("$%s Okteto Platform variable is being overridden by a local environment variable with the same name.", v.Name)
```

Comment:
```suggestion
				oktetoLog.Warning("$%s Okteto variable is being overridden by a local environment variable with the same name.", v.Name)
```
We don't have a concept for "Okteto Platform variable"

Reply:
@pchico83 we have been using this term internally in the dev team to indicate User plus Cluster variables. Do you have a better alternative in mind? Or how would you change the copy? cc @codyjlandstrom

Reply:
"Admin variables" is the name that we've been using across the [documentation](https://www.okteto.com/docs/admin/dashboard/#variables).
This [doc](https://www.okteto.com/docs/admin/dashboard/#variables) helps to document the hierarchy of the variables if that's helpful.
Lets proceed with "Admin Variables"

Reply:
Seperately, we should rename the "Variables" section in the admin UI to be "Admin Variables" for clarity

Reply:
@rlamana could you create an issue to track the UI changes?

Reply:
Just to clarify, we are talking about changing "Variables" to "Admin Variables" in the Admin section of the UI, right?

Reply:
Issue created here: https://okteto.atlassian.net/browse/DEV-313

Reply:
Does "Admin Variables" include the "User Variables"? Why do we call them "Admin" even if I am just a user and I can add my own vars? 🤔  @codyjlandstrom

Reply:
We call them admin because only admin can define those and they have a cluster wide scope. As a user, the only thing you need to now is that you can override Admin variables. We should inform about this through the docs and the UI.

Reply:
@codyjlandstrom @pchico83 @ifbyol could we list in the "user variables" the variables that are already defined and that the user can override? Would that be useful? A priori I think it could help a lot. Not sure if there are security implications, we could only show the name but not the values, as I think we already do today...

Reply:
@codyjlandstrom in the docs we have the concept of "User Variables" and "Admin Variables" they get merged together at this point so I cannot differenciate and let the user know which one is it. That's wht I am trying to find a common name for those. Thoughts?

Reply:
Here is that document on variable preference: https://www.okteto.com/docs/core/okteto-variables/#user-defined-variables.
@andreafalzetti Since I'm just getting up to speed with this feature. The "Platform Variable" in reference can only be created by the admin?

Reply:
I think the closer term to Okteto Platform Variables is "Okteto Variables":
https://www.okteto.com/docs/core/okteto-variables/
I would go with:
```
Okteto Variable '%s' is overridden by a local environment variable with the same name
```

Reply:
We are talking different things in the same thread and we might be confusing the original purpose of the comment.

For context, CLI doesn't know if they are Admin or User variables. CLI just gets the variables that have to be taken into account. For example, If a User variable overrides an Admin Variable, CLI will just know the User one.

> The "Platform Variable" in reference can only be created by the admin?

Replying to @codyjlandstrom. "Platform Variable" as @andreafalzetti  called it is any Variable defined through the UI (Admin or User). 

Taking into account that, I agree with @pchico83 's last comment about just call them `Okteto Variable`. Let's discuss in other thread/place the other changes mentioned here to avoid confusions